.editorconfig

```diff
@@ -13,5 +13,5 @@ end_of_line = lf
 insert_final_newline = true
 trim_trailing_whitespace = true
 translate_tabs_to_spaces = true
```

Comment:
What is this rule? it doesn't show on up https://github.com/editorconfig/editorconfig/wiki/EditorConfig-Properties

Reply:
May be either a deprecated feature - or a Sublime Text specific thing (see first sentence after code): https://github.com/sindresorhus/editorconfig-sublime/issues/12

Automatically converts tabs to spaces on new files. Probably a horrible idea all around anyway. Good catch. ;)

Removed.